tests/system/Helpers/FormHelperTest.php

```diff
@@ -612,7 +612,7 @@ public function testFormFieldsetClose()
     public function testFormCheckbox()
     {
         $expected = <<<EOH
-            <input type="checkbox" name="newsletter" value="accept" checked="checked" />\n
+            <input type="checkbox" name="newsletter" value="accept" checked="checked">\n
             EOH;
         $this->assertSame($expected, form_checkbox('newsletter', 'accept', true));
```

Comment:
Can you add a test case for XHTML?

Reply:
I added test case for XHTML in void element to directy related with `solidus`

Reply:
`testFormInputXHTML()` was added. Thanks!